docs/local_ollama.md

```diff
@@ -4,16 +4,32 @@ You may want to save costs by developing against a local LLM server, such as
 [llamafile](https://github.com/Mozilla-Ocho/llamafile/). Note that a local LLM
 will generally be slower and not as sophisticated.
 
-Once you've got your local LLM running and serving an OpenAI-compatible endpoint, define `LOCAL_OPENAI_ENDPOINT` in your `.env` file.
+Once you've got your local LLM running and serving an OpenAI-compatible endpoint, define `LOCAL_MODELS_ENDPOINT` in your `.env` file.
```

Comment:
Hm, why is it LOCAL_MODELS?

Reply:
Looks like another difference between this and the openai-chat-app-quickstart sample. That one uses LOCAL_MODELS_ENDPOINT. I like that name more than LOCAL_OPENAI_ENDPOINT, but your call if you want to align the two repos. Happy to edit this PR either way.

Reply:
Ohh.. Hmm. I like LOCAL_OPENAI in terms of it being an openai-compatible endpoint, but LOCAL_MODELS also nice in that youre using local models. I'm fine with LOCAL_MODELS change, please modify rest of repo accordingly.